streamwebs_frontend/streamwebs/templates/streamwebs/resources.html

```diff
@@ -0,0 +1,47 @@
+{% extends "streamwebs/base.html" %}
+
+{% block content %}
+<div class="container">
+  <h3>
+    <strong>
+      StreamWebs offers a number of resources to get you started in the field.
+    </strong>
+  </h3>
+  <p><a href="data-sheets">DATA SHEETS</a>:
+  </br>
+    These field data forms support a number of watershed field analyses, including
+    water quality sampling, macroinvertebrate sampling, and photo point
+    monitoring. Click on the link to see all of the data forms you can
+    download.
+  </p>
+  <p><a href="http://seagrant.oregonstate.edu/education/kits">EDUCATOR KITS</a>:
+  </br>
+    Streamwebs has education kits available for check-out at no charge. These
+    kits can be checked out by teachers, watershed educators, or other non-profit
+    groups or organizations in Oregon. Kits are available for pick-up in
+    Corvallis or Newport.
+  </p>
+  <p><a href="curriculum-guides/">CURRICULUM PUBLICATIONS</a>:
+  </br>
+    In this section, you will be able to download curriculum to use in the field
+    and the classroom, including Salmon Watch, Student Stewardship Project Guide,
+    and 1000 Drops.
+  </p>
+  <p><a href="http://www.education.noaa.gov">NOAA EDUCATOR RESOURCES:</a>
+  </br>
+    NOAA has a number of education resources that can support watershed
+    investigations both in the field and classroom. Examples include:
+    data resources, ocean acidification activities, and global systems activities.
+  </p>
+  <p><a href="tutorial-videos">TUTORIAL VIDEOS</a>:
+  </br>
+    Here you will find a number of video tutorials covering field protocols for
+    using the Vernier LabQuest and water quality sensors in the field, in addition
+    to a field tutorial for conducting photo point monitoring.
+  </p>
+  <p><a href="glossary">GLOSSARY</a>:
+  </br>
+    The StreamWebs glossary contains some of the most important watershed terms
+    and concepts.
+  </p>
+{% endblock %}
```

Comment:
Always make sure you closed all your tags! (You left the div tag at the top unclosed.)